components/channel_header_dropdown/channel_header_dropdown_items.tsx

```diff
@@ -137,7 +139,6 @@ export default class ChannelHeaderDropdown extends React.PureComponent {
                         user={user}
                         channel={channel}
                         isMuted={isMuted}
-                        isArchived={isArchived}
```

Comment:
Is not required anymore?

Reply:
I couldn't determine `isArchived` is required.
I guessed `isArchived` is used to hide toggle mute menu item when channel is archived so I fixed this in 98696dff5e7385c05d7d139c4841436e2a79b82a.
But as I mentioned above, this is out of scope of this PR.
I removed `isArvhiced`.

Reply:
yes it seems that this is dangling props, usage of it is not found in `MenuItemToggleMuteChannel`